app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -1,5 +1,69 @@
 module Api
   class ConversionHostsController < BaseController
     include Subcollections::Tags
+
+    # Enable the conversion host role by installing the conversion host module
+    # and running the conversion host playbook that enables it.
+    #
+    # POST /api/conversion_hosts/:id { "action": "enable" }
+    #
+    def enable_resource(type, id, data)
+      conversion_host = resource_search(id, type, collection_class(type))
+
+      if conversion_host.eligible?
```

Comment:
Probably not suitable to check `eligible?` here
1) API server could be running in a different zone (firewall) from where the provider is 
2) that method check how many migration task the `conversion_host` already has. sounds like an chicken and egg

Reply:
Ok, I will skip it for now. Perhaps we'll come back to it.